Client/Application/main.swift

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+private var appDelegate: AppDelegate.Type
+
+if AppConstants.IsRunningTest {
```

Comment:
I think it could be nice if we had an `enum` containing the Build or Channel type. What do you think?